web/packages/teleport/src/services/integrations/types.ts

```diff
@@ -424,6 +426,8 @@ export type UserTask = {
 
 // UserTaskDetail contains all the details for a User Task.
 export type UserTaskDetail = UserTask & {
+  // title is the issue title.
+  title: string;
```

Comment:
Isn't this superfluous here since `UserTaskDetail` already reuses the properties from `UserTask`?